cookbook/models.py

```diff
@@ -194,6 +205,7 @@ class Recipe(models.Model):
     working_time = models.IntegerField(default=0)
     waiting_time = models.IntegerField(default=0)
     internal = models.BooleanField(default=False)
+    nutrition = models.ForeignKey(NutritionInformation, blank=True, null=True, on_delete=models.PROTECT)
```

Comment:
why is this PROTECT ? shouldn't it be CASACE (if the recipe is delete, delete the nutritional info as well ?) maybe i am mixing something up, dont have time to test it atm

Reply:
Ah, sure! Mix it up somehow